services/121-service/src/registration/dto/send-custom-text.dto.ts

```diff
@@ -1,26 +1,21 @@
 import { ApiProperty } from '@nestjs/swagger';
-import {
-  IsBoolean,
-  IsOptional,
-  IsString,
-  MinLength,
-  ValidateIf,
-} from 'class-validator';
+import { IsBoolean, IsOptional, ValidateIf } from 'class-validator';
+import { IsNotBothEmpty } from '../validators/is-not-both-empty.class.validator';
+import { IsNotBothPresent } from '../validators/is-not-both-present.class.validator';
 
 export class SendCustomTextDto {
   @ApiProperty({
     example: 'Your voucher can be picked up at the location',
   })
-  @IsString()
-  @MinLength(1)
-  @ValidateIf((o) => !o.skipMessageValidation && !o.messageTemplateKey)
+  @IsNotBothPresent('messageTemplateKey')
+  @IsNotBothEmpty('messageTemplateKey')
```

Comment:
Rather than using two validators, maybe it would be better to have something like `OnlyOneIsPresent` that combines the two. Ideally this somehow reuses `IsNotBothPresent`

Reply:
Is see your point. But for decorators seperating them is a bit better because it fits more with the style of how the default decorators are setup.   

`@IsOptional()
 @IsBoolean()`

Could also be combined in `IsOptionalBoolean`

Reply:
I see your point actually. Let's leave this as-is